lib/language.js

```diff
@@ -4811,138 +4858,161 @@ function init() {
       ,it: 'Profilo'
       ,ro: 'Profil'
+      ,sv: 'Profil'
       ,nb: 'Profil'
       ,fi: 'Profiili'
       }
     ,'General profile settings' : {
       cs: 'Obecná nastavení profilu'
       ,ro: 'Setări generale profil'
+      ,sv: 'Allmän profilinställning'
       ,nb: 'Profilinstillinger'
       ,fi: 'Yleiset profiiliasetukset'
       }
     ,'Title' : {
       cs: 'Název'
       ,ro: 'Titlu'
+      ,sv: 'Titel'
       ,nb: 'Tittel'
       ,fi: 'Otsikko'
       }
     ,'Database records' : {
       cs: 'Záznamy v databázi'
       ,ro: 'Înregistrări'
+      ,sv: 'Databashändelser'
       ,nb: 'Databaseverdier'
       ,fi: 'Tietokantamerkintöjä'
       }
     ,'Add new record' : {
       cs: 'Přidat nový záznam'
       ,ro: 'Adaugă înregistrare nouă'
+      ,sv: 'Lägg till ny händelse'
       ,nb: 'Legg til ny rad'
       ,fi: 'Lisää uusi merkintä'
       }
     ,'Remove this record' : {
       cs: 'Vymazat tento záznam'
       ,ro: 'Șterge această înregistrare'
+      ,sv: 'Ta bort denna händelse'
       ,nb: 'Fjern denne raden'
       ,fi: 'Poista tämä merkintä'
       }
     ,'Clone this record to new' : {
       cs: 'Zkopíruj tento záznam do nového'
       ,ro: 'Duplică această înregistrare'
+      ,sv: 'Kopiera denna händelse till ny'
       ,nb: 'Kopier til ny rad'
       ,fi: 'Kopioi tämä merkintä uudeksi'
       }
     ,'Record valid from' : {
       cs: 'Záznam platný od'
       ,ro: 'Înregistare validă de la'
+      ,sv: 'Händelse giltig från'
       ,nb: 'Rad gyldig fra'
       ,fi: 'Merkintä voimassa alkaen'
       }
     ,'Stored profiles' : {
       cs: 'Uložené profily'
       ,ro: 'Profile salvate'
+      ,sv: 'Lagrad profil'
       ,nb: 'Lagrede profiler'
       ,fi: 'Tallennetut profiilit'
       }
     ,'Timezone' : {
       cs: 'Časová zóna'
       ,ro: 'Fus orar'
+      ,sv: 'Tidszon'
       ,nb: 'Tidssone'
       ,fi: 'Aikavyöhyke'
       }
     ,'Duration of Insulin Activity (DIA)' : {
       cs: 'Doba působnosti inzulínu (DIA)'
       ,ro: 'Durata de acțiune a insulinei'
+      ,sv: 'verkningstid för insulin (DIA)'
       ,nb: 'Insulin varighet'
       ,fi: 'Insuliinin vaikutusaika (DIA)'
       }
     ,'Represents the typical duration over which insulin takes effect.  Varies per patient and per insulin type. Typically 3-4 hours for most pumped insulin and most patients. Sometimes also called insulin lifetime.' : {
       cs: 'Představuje typickou dobu, po kterou inzulín působí. Bývá různá podle pacienta a inzulínu. Typicky 3-4 hodiny pro pacienty s pumpou.'
       ,ro: 'Reprezintă durata tipică pentru care insulina are efect. Este diferită la fiecare pacient și pentru fiecare tip de insulină'
+      ,sv: 'Beskriver under hur lång tid insulinet verkar' Varierar mellan patienter. Typisk varaktighet 3-4 timmar.'
       ,nb: 'Representerer typisk insulinvarighet. Varierer per pasient og per insulin type. Vanligvis 3-4 timer for de fleste typer insulin og de fleste pasientene. Noen ganger også kalt insulinlevetid.'
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '{' and instead saw 'de'.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1367486668/issues/source?bid=2452379&fileBranchId=2976078#l4939)